src/EnergyPlus/OutputProcessor.cc

```diff
@@ -6468,26 +6466,37 @@ void UpdateMeterReporting(EnergyPlusData &state)
     }
 
     // Helper lambda to locate a meter index from its name. Returns a negative value if not found
-    auto findMeterIndexFromMeterName = [&state](std::string const &name) -> int {
-        auto &op(state.dataOutputProcessor);
+    auto setupMeterFromMeterName =
+        [&state](std::string &name, std::string const &freqString, bool MeterFileOnlyIndicator, bool CumulativeIndicator) -> bool {
```

Comment:
Returning true instead of the index, I guess now that the caller code won't need to set it up anymore, they don't need it, just a bool to signify success or failure.  OK.